searchlib/src/vespa/searchlib/engine/request.h

```diff
@@ -30,7 +30,11 @@ class Request
 
     bool should_drop_sort_data() const;
 
-    Request & setTraceLevel(uint32_t level) { _trace.setLevel(level); return *this; }
+    Request & setTraceLevel(uint32_t level, uint32_t minLevel) {
```

Comment:
if the minimum level is hard coded anyways, why not just let the trace class control/own it, instead of the packet converter?

Reply:
I guess the correct place is the matchengine.

Reply:
Moved to matchengine